### PR TITLE
add uiautomator and uiautomator2 driver docs

### PR DESCRIPTION
**Pull request by jlipps**
consolidate all setup instructions into these docs

### CODE REVIEW

**Review by dpgraham — APPROVED** [review]
Looks good.

File: docs/en/drivers/android-uiautomator2.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+The way to start a session using the UiAutomator driver is to include the
+`automationName` [capability](#TODO) in your [new session request](#TODO), with
+the value `UiAutomator2`. Of course, you must also include appropriate
+`platformName` (=`Android`), `platformVersion`, `deviceName`, and `app`
+capabilities, at a minimum.
+
+### Capabilities
+
+The UiAutomator driver supports a number of standard [Appium
```
Comment (imurchie) [inline docs/en/drivers/android-uiautomator2.md]: UiAutomator2

File: docs/en/drivers/android-uiautomator2.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+capabilities](/docs/en/writing-running-appium/caps.md), but has an additional
+set of capabilities that modulate the behavior of the driver. These can be
+found currently at [Android
+section](/docs/en/writing-running-appium/caps.md#android-only) of the
+aforementioned doc.
+
+For web tests, to automate Chrome instead of your own application, leave the
+`app` capability empty and instead set the `browserName` capability to
+`Chrome`. Note that you are responsible for ensuring that Chrome is on the
+emulator/device, and that it is of a version compatible with Chromedriver.
```
Comment (imurchie) [inline docs/en/drivers/android-uiautomator2.md]: Could link to the chromedriver doc here, for compatible versions.

File: docs/en/drivers/android-uiautomator2.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+
+### Commands
+
+To see the various commands Appium supports, and specifically for information
+on how the commands map to behaviors for the UiAutomator driver, see the [API
```
Comment (imurchie) [inline docs/en/drivers/android-uiautomator2.md]: UiAutomator2

File: docs/en/drivers/android-uiautomator2.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+devices, other than these simple requirements:
+
+* Ensure that [Developer
+  mode](https://developer.android.com/studio/debug/dev-options.html) is turned
+  on for the device.
+* Ensure that the device is connected via USB to the Appium host, and can be
+  seen by [ADB](https://developer.android.com/studio/command-line/adb.html)
+  (run `adb devices` to make sure).
+* Ensure that "Verify Apps" in settings is disabled, to allow Appium's helper
+* apps to function without manual intervention.
```
Comment (imurchie) [inline docs/en/drivers/android-uiautomator2.md]: Extra `*`?

**Review by imurchie — COMMENTED** [review]
A few small changes. Nothing substantive.

**Comment by jlipps** [comment]
@imurchie good catches; changes made

**Review by imurchie — APPROVED** [review]

**Pull request merged**